.github/workflows/ci.yml

```diff
@@ -109,8 +109,8 @@ jobs:
         #
         # ...to:
         #
-        #   toolchain: $ {{ ./cargo.sh --version matrix.toolchain }} # hypothetical syntax
-        ZC_TOOLCHAIN="$(./cargo.sh --version ${{ matrix.toolchain }})"
+        #   toolchain: $ {{ cargo run -p cargos --version matrix.toolchain }} # hypothetical syntax
```

Comment:
I'm not familiar with Windows enough to know: Is there a way that we could keep a script that would be compatible with both Windows and POSIX, and would just be a thin wrapper around `cargo run -p cargos`? The latter is more verbose, which isn't great for the command we run the most frequently during development.

Reply:
We could just publish this as a separate package and `cargo install` it. Another option would be to leave the `cargo.sh` file in place and make it a thin wrapper over `cargo run -p cargos`.